go.sum

```diff
@@ -631,8 +631,8 @@ github.com/zclconf/go-cty v1.13.2 h1:4GvrUxe/QUDYuJKAav4EYqdM47/kZa672LwmXFmEKT0
 github.com/zclconf/go-cty v1.13.2/go.mod h1:YKQzy/7pZ7iq2jNFzy5go57xdxdWoLLpaEp4u238AE0=
 github.com/zclconf/go-cty-debug v0.0.0-20191215020915-b22d67c1ba0b/go.mod h1:ZRKQfBXbGkpdV6QMzT3rU1kSTAnfu1dO8dPKjYprgj8=
 go.mongodb.org/atlas v0.12.0/go.mod h1:wVCnHcm/7/IfTjEB6K8K35PLG70yGz8BdkRwX0oK9/M=
```

Comment:
🤔 I wonder why there's a reference to such an old version

Reply:
Looking at `go mod graph` and I see that `go.mongodb.org/realm@v0.1.0` use `go.mongodb.org/atlas@v0.12.0`

```bash
go.mongodb.org/realm@v0.1.0 github.com/google/go-querystring@v1.1.0
go.mongodb.org/realm@v0.1.0 github.com/kr/pretty@v0.1.0
go.mongodb.org/realm@v0.1.0 github.com/openlyinc/pointy@v1.1.2
go.mongodb.org/realm@v0.1.0 github.com/stretchr/testify@v1.4.0
go.mongodb.org/realm@v0.1.0 go.mongodb.org/atlas@v0.12.0
```